app/traffic_cloud_utils/statusHelper.py

```diff
@@ -70,9 +70,12 @@ def mark_all_failed():
         identifiers = get_all_projects()
         for identifier in identifiers:
             status = StatusHelper.get_status(identifier)
-            for (k, v) in status.iteritems():
-                if v == Status.Flag.IN_PROGRESS:
-                    StatusHelper.set_status(identifier, k, Status.Flag.FAILURE)
+            if status:
+                for (k, v) in status.iteritems():
+                    if v == Status.Flag.IN_PROGRESS:
+                        StatusHelper.set_status(identifier, k, Status.Flag.FAILURE)
+            else:
+                print "Error: Could not get CFG file, not clearing this project"
```

Comment:
Instead of "not clearing this project" say "could not mark project status failure flags for project {}".format(identifier)